app/helpers/routing_helper.rb

```diff
@@ -2,37 +2,40 @@
 require 'json'
 
 module RoutingHelper
-  def self.seconds_to_minsec(sec)
+  def self.format_seconds_as_minsec(sec)
     format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60, seconds: sec % 60)
   end
 
   def self.valid_coordinates(coordinates)
     return false if coordinates.blank?
 
-    regex = /^[0-9]{1,2}(\.[0-9]{1,6})?,[0-9]{1,2}(\.[0-9]{1,6})?$/
+    regex = /^-?\d{1,2}(\.\d{1,6})?,-?\d{1,2}(\.\d{1,6})?$/
     coordinates.match(regex)
   end
 
+  def self.routing_url(start, destination)
+    (start_lat, start_long) = start.split(",")
+    (dest_lat, dest_long) = destination.split(",")
+    "http://routing.openstreetmap.de/routed-foot/route/v1/driving/#{start_long},#{start_lat};#{dest_long},#{dest_lat}?overview=full&geometries=geojson"
+  end
+
   def self.calculate_route(start, destination)
-    return nil unless valid_coordinates(start) && valid_coordinates(destination) # No route requested
+    return unless valid_coordinates(start) && valid_coordinates(destination)
```

Comment:
Since this is now done in `index`, we don't need it here anymore and can fail hard, right?

Reply:
Well we would need it here for our tests that call this method with invalid inputs. It might also be good to have them here in case another route wants to use this method.

Reply:
Can't you assert that an exception is thrown in the test?

Reply:
We could test for that. I still think it's neat to check here (in this case again).
Any thought from someone else maybe?

Reply:
@christian-helms What do you think?

Reply:
I agree with Julian. Besides, it documents the desired input format pretty well. If someone wants to use this method, the person just has to take a look into the validator method and does not have to bother looking into the API documentation.